src/Compilers/CSharp/Portable/Symbols/OverriddenOrHiddenMembersResult.cs

```diff
@@ -52,6 +53,37 @@ internal sealed class OverriddenOrHiddenMembersResult
             }
         }
 
+        internal static Symbol GetOverriddenMember(Symbol substitutedOverridingMember, Symbol overriddenByDefinitionMember)
```

Comment:
I think there is a benefit in having both methods (this one and the one below it) to have the same name, discoverability, etc.